utils/obsproc/Ghrsst2Ioda.h

```diff
@@ -28,10 +28,21 @@ namespace gdasapp {
     gdasapp::IodaVars providerToIodaVars(const std::string fileName) final {
       oops::Log::info() << "Processing files provided by GHRSST" << std::endl;
 
+      // Get the sst bounds from the configuration
+      std::string sstUnits;
+      fullConfig_.get("bounds.units", sstUnits);
+      float sstMin;
+      fullConfig_.get("bounds.min", sstMin);
+      float sstMax;
+      fullConfig_.get("bounds.max", sstMax);
+      if ( sstUnits == "C" ) {
```

Comment:
Are these increments in the right direction? If the units are not definitively known in advance, how safe is the assumption that not-"C" equals Kelvin?

Reply:
Probably lines below makes sst "C " @AndrewEichmann-NOAA 
`
sst[i][j] = static_cast<float>(sstObsVal[index]) * sstScaleFactor + sstOffSet
            - static_cast<float>(sstObsBias[index]) * biasScaleFactor;
`

Reply:
No @apchoiCMD , it doesn't. The output is in K.

Reply:
@guillaumevernieres Yes, thanks for pointing me out.

Reply:
@AndrewEichmann-NOAA , maybe the variable name is confusing. `sstUnits` is meant to be the unit of the bounds read from the yaml.

Reply:
Tentatively assuming you are OK with the explanation above @AndrewEichmann-NOAA and marking the conversation as "resolved ".